unity-client/Assets/Scripts/MainScripts/DCL/Components/BuilderInWorld/DCLVisibleOnEdit.cs

```diff
@@ -0,0 +1,39 @@
+using DCL;
+using DCL.Components;
+using DCL.Controllers;
+using DCL.Models;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+/// <summary>
+/// This component describes the visibility of the Entity in the builder in world.
+/// Builder in World send a message to kernel to change the value of this component in order to show/hide it
+/// </summary>
+public class DCLVisibleOnEdit : BaseDisposable
+{
+    [System.Serializable]
+    public class Model
+    {
+        public bool value;
```

Comment:
Same as before, if possible, rename to something more useful.

Reply:
Same as before. The problem is that the kernel part name is "value". Since we get the model and convert it to JSON and send to kernel, we can't change the name without changing it on the kernel side

Reply:
Renamed to `isVisible ` @nchamo let me know when you change the name so I can push the changes!